src/app/(dashboard)/repositories/[repositoryId]/page.tsx

```diff
@@ -20,6 +20,9 @@ import { LoadingCircle } from "@/components/navigation/loading";
 import { TextGenerateEffect } from "@/components/ui/text-generate-effect";
```

Comment:
Side not, not related to this, The repo page: We should have a empty state for that page